pkg/operator/certrotation/target.go

```diff
@@ -104,7 +111,7 @@ func (c RotatedSelfSignedCertKeySecret) ensureTargetCertKeyPair(ctx context.Cont
 
 	if reason := needNewTargetCertKeyPair(targetCertKeyPairSecret.Annotations, signingCertKeyPair, caBundleCerts, c.Refresh, c.RefreshOnlyWhenExpired); len(reason) > 0 {
```

Comment:
Needs the `ensureTLSMetadata` block above this to handle the update case, right?

Reply:
done